<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadsListWidget</class>
 <widget class="QWidget" name="ThreadsListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,80">
   <item alignment="Qt::AlignRight">
    <widget class="QLabel" name="ProfilePictureLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="ThreadSubjectLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Thread Subject</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ThreadTagsLabel">
       <property name="text">
        <string>Thread Tags</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="ThreadCreatorName">
       <property name="text">
        <string>Name of Thread Creator</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
